ducc/src/tests/object.rs

```diff
@@ -35,6 +37,40 @@ fn set_get() {
     assert_eq!(object.get::<_, String>(123).unwrap(), "a");
 }
 
+#[test]
+fn define_prop() {
+    let mut ducc = Ducc::new();
+    let object = ducc.create_object();
+
+    object.define_prop("a", PropertyDescriptor {
+        value: Some(123),
+        ..Default::default()
+    }).unwrap();
+    assert_eq!(object.get::<_, i8>("a").unwrap(), 123);
+
+    let get = ducc.create_function(|inv| Ok(24));
+    object.define_prop("b", PropertyDescriptor::<()> {
+        get: Some(get),
+        ..Default::default()
+    }).unwrap();
+    assert_eq!(object.get::<_, i8>("b").unwrap(), 24);
+
+    let mut v = Arc::new(Mutex::new(0));
```

Comment:
This is a bit convoluted, maybe there is a more compact and Rusty way of achieving this?

Reply:
For the sake of the test, I would just set/check a value on `inv.ducc.globals()` instead of using a Rust-land variable.

(By the way, I'm guessing you had to use `Arc<Mutex<...>>` here instead of `Rc<RefCell<...>>` is because `Ducc::create_function` requires `Send`... I think this bound is erroneous. I just fixed a similar problem for MiniV8. I'll address it for ducc soon.)